botstrap.py

```diff
@@ -40,14 +46,61 @@ def __init__(self):
         super().__init__(
             base_url="https://discord.com/api/v10",
             headers={"Authorization": f"Bot {BOT_TOKEN}"},
-            event_hooks={"response": [self._raise_for_status]}
+            event_hooks={"response": [self._raise_for_status]},
         )
 
     @staticmethod
     def _raise_for_status(response: Response) -> None:
         response.raise_for_status()
 
 
+def upgrade_server_to_community_if_necessary(
+        guild_id: int | str,
+        rules_channel_id_: int | str,
+        announcements_channel_id_: int | str,
```

Comment:
Just a nitpick, but here and in other places you're adding an underscore after "id". It's only necessary for variables named `id_` since `id` is a built-in.

Reply:
Actually that was just because they're shadowing variables from the module's namespace and were just bugging me in Pycharm.

Maybe I chose poor names, do you want me to change them?

Reply:
It's fine either way, I was just puzzled.